src/main/java/com/vendor/caterer/dto/CategoryUpdateRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.vendor.caterer.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@Data
+public class CategoryUpdateRequest {
+
+    private String name;
+    private String description;
+    private UUID catererId;
```

Comment:
Why you need catereId in Update request, update request should only contain properties that can be updated no need to update catererId.

Reply:
DONE